src/frequently-asked-questions.md

```diff
@@ -1 +1,238 @@
 # Frequently Asked Questions
+
+This chapter addresses common questions and challenges that developers encounter when working with Rust on ESP chips. Whether you're setting up your development environment, optimizing your code, or looking to simulate your projects, you'll find practical solutions and best practices here.
+
+## Editor/IDE
+
+When using `esp-generate`, you can automatically configure recommended settings and extensions for both VS Code and Helix editors.
+
+### Visual Studio Code
+
+VS Code provides great support for Rust ESP development with the right extensions and settings.
+
+#### Recommended Extensions
+
+- [`rust-analyzer`](https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer) - Provides intelligent code completion, go to definition, and other IDE features
```

Comment:
Maybe we can mention [mksettings](https://github.com/bjoernQ/mksettings) here? But probably it's not that useful for users developing for _just_ one target

Reply:
Probably if `esp-generate` fails to create a working `.vscode/settings.json` that would be something we need to fix